metriq_gym/benchmarks/__init__.py

```diff
@@ -0,0 +1,20 @@
+from metriq_gym.benchmarks.benchmark import Benchmark, BenchmarkData
```

Comment:
Should we put all of this in the `benchmarks.py` file? Kind of like how we have the provider components in `provider.py` (and not in `__init__.py`).